src/system/baremetalpi/kernel.cpp

```diff
@@ -181,25 +181,14 @@ static System systemInterface =
 	.updateConfig = updateConfig,
 };
 
-u32 rgbaToBgra(u32 u){
-	u8 r = u & 0xFF;
-	u8 g = (u >> 8) & 0xff;
-	u8 b = (u >> 16) & 0xff;
-	return (b & 0xFF)      
-					| ((g & 0xFF) << 8)
-					| ((r & 0xFF)   << 16)
-					| (0xFF << 24);
-}
-
 void screenCopy(CScreenDevice* screen, u32* ts)
 {
 	u32 pitch = screen->GetPitch();
 	u32* buf = screen->GetBuffer();
 	for (int y = 0; y<136;y++)
-	for (int x = 0; x<240;x++)
 	{
-		u32 p = ts[(y+4)*(8+240+8)+(x+8)];
-		buf[pitch*y + x] = rgbaToBgra(p);
+		u32 *line = ts + ((y+TIC80_OFFSET_TOP)*(TIC80_FULLWIDTH) + TIC80_OFFSET_LEFT);
+		memcpy(buf + (pitch * y), line, 240 * 4);
```

Comment:
pls use `TIC80_WIDTH` `TIC80_HEIGHT` instead `240` `136` integers here

Reply:
Done. Are there any build instructions for baremetalpi? Not sure how to check if it still compiles...

Reply:
Here are the build instructions https://github.com/msx80/BareMetalTic80#how-to-build.
Also, we are going to add the baremetal build to the CI server and it will be checked automatically.